editor/app/crashes/[record_locator]/page.tsx

```diff
@@ -69,6 +69,15 @@ export default function CrashDetailsPage({
     await refetch();
   }, [refetch]);
 
+  // When data is loaded or updated this sets the title of the page inside the HTML head element
+  useEffect(() => {
+    if (!!data) {
+      document.title = `${data[0].cris_crash_id} - ${data[0].address_primary ? data[0].address_primary : ""} ${
+        data[0].address_secondary ? "& " + data[0].address_secondary : ""
+      }`;
+    }
+  }, [data]);
```

Comment:
looks great! since we're doing this address formulation in multiple places, it might be nice to move to a helper.

Reply:
Really, I'd like to fix this at the DB level with a generated field that spits out the concatenated address - out of scope for sure though

Reply:
oh, i was referring to the address concat, but yeah maybe a component for the title setter makes sense too, eh?

Reply:
cool i put the address concat helper in the `formatters.ts`, i think that makes sense to keep it there?